.github/PULL_REQUEST_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,33 @@
+## Summary
+
+Brief description of the documentation update and its purpose.
+
+## Areas Affected
+
+List the sections or pages of the documentation that are updated.
+
+## Details
+
+Provide detailed information about the changes made to the documentation.
+
+## Motivation
+
+Explain why these documentation changes are necessary.
+
+## Review Checklist
```

Comment:
This section is `Review Checklist` where it's `Checklist` in the other templates, is that an intentional difference?